tutorials/todo/src/main.sw

```diff
@@ -0,0 +1,51 @@
+contract;
+
+dep abi;
+dep tasks;
+
+use std::storage::StorageMap;
+use abi::Todo;
+use tasks::Task;
+
+storage {
+    tasks: StorageMap<u64, Task>,
```

Comment:
Probably want to add an owner which is the address of the wallet that has been generated and add logic to make sure that they are the only one who can manipulate their task list